### PR TITLE
Add useful example to `http options` documentation

### PR DESCRIPTION
**Pull request by bgmort**
This PR adds a more real-world example of how the new http options is used, as well as some additional information in the description.

### CODE REVIEW

File: crates/nu-command/src/network/http/options.rs [hunk 1]
```diff
@@ -97,18 +97,8 @@ impl Command for SubCommand {
                 result: None,
             },
             Example {
-                description: "Get options from example.com, with username and password",
-                example: "http options -u myuser -p mypass https://www.example.com",
-                result: None,
-            },
```
Comment (fdncred) [inline crates/nu-command/src/network/http/options.rs:103]: I don't think it does any harm to have other generic examples here. So, I'd leave these.

File: crates/nu-command/src/network/http/options.rs [hunk 2]
```diff
@@ -97,18 +97,8 @@ impl Command for SubCommand {
...
-            Example {
-                description: "Get options from example.com, with custom header",
-                example: "http options -H [my-header-key my-header-value] https://www.example.com",
-                result: None,
-            },
-            Example {
-                description: "Get options from example.com, with custom headers",
-                example: "http options -H [my-header-key-A my-header-value-A my-header-key-B my-header-value-B] https://www.example.com",
+                description: "Simulate a browser cross-origin preflight request from www.example.com to media.example.com",
+                example: "http options https://media.example.com/api/ -H [Origin https://www.example.com Access-Control-Request-Headers \"Content-Type, X-Custom-Header\" Access-Control-Request-Method GET]",
```
Comment (fdncred) [inline crates/nu-command/src/network/http/options.rs:101]: Is this actually supposed to return something? It doesn't for me.

**Comment by fdncred** [comment]
This is the right place to add this stuff, versus the docs. Thanks for taking the time to add it. I just had a couple comments above. Appreciate your help!

File: crates/nu-command/src/network/http/options.rs (hunk 2 above)
Comment (bgmort) [reply crates/nu-command/src/network/http/options.rs:101]: No, media.example.com is not a real endpoint.

Comment (fdncred) [reply crates/nu-command/src/network/http/options.rs:101]: ok, thanks for the info.

File: crates/nu-command/src/network/http/options.rs (hunk 1 above)
Comment (bgmort) [reply crates/nu-command/src/network/http/options.rs]: Added them back in.

Comment (fdncred) [reply crates/nu-command/src/network/http/options.rs]: Thanks!

**Review by fdncred — APPROVED** [review]
Appreciate your help here1

**Review by amtoine — APPROVED** [review]
more examples? loving it!

thanks @bgmort :partying_face:

**Pull request merged**